packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { _formatString } from '@internal/acs-ui-common';
+import { RemoteParticipantState } from '@internal/calling-stateful-client';
+import { useEffect, useMemo, useState } from 'react';
+import { useLocale } from '../../localization';
+import { CallAdapterState } from '../adapter';
+import { useSelector } from '../hooks/useSelector';
+
+/**
+ * Custom selector for this hook to retrieve all the participants that are currently
+ * connected to the call.
+ */
+const getRemoteParticipantsConnectedSelector = (callState: CallAdapterState): RemoteParticipantState[] => {
```

Comment:
Actually make this a selector. See https://github.com/Azure/communication-ui-library/blob/main/docs/architecture/WritingSelectors.md